models/model.js

```diff
@@ -143,4 +143,22 @@ exports.addComment = (article_id, username, body) => {
 
       
 
+      exports.removeComment = (comment_id) => {
 
+        querySQL =  
+        `DELETE FROM comments 
+        WHERE comment_id = $1 
+        RETURNING *;`
+        
+        const queryValue = [comment_id]
+
+        return db
+          .query(querySQL, queryValue)
+          .then(({ rows }) => {
+            if (rows.length === 0) {
+              return Promise.reject(
+                {status: 404, msg: "not found"});
+            }
+            return rows;
```

Comment:
We shouldn't be sending anything back as a response, fine to check the rows for error handling but then shouldn't send it back.